node/config/doc_gen.go

```diff
@@ -239,6 +239,13 @@ see https://docs.filecoin.io/mine/lotus/miner-configuration/#using-filters-for-f
 
 			Comment: `Whether to do commp on the Boost node (local) or on the Sealer (remote)`,
 		},
+		{
+			Name: "HTTPRetrievalMultiAddr",
+			Type: "string",
+
+			Comment: `The public multi-address for retrieving deals with booster-http.
+Note: Must be in multiaddr format, eg /dns6/foo.com/tcp/80/http`,
```

Comment:
Folks will likely want to https this ( so they can take advantage of various cache-fronting offerings ). Probably worth changing to 

```suggestion
Note: Must be in multiaddr format, eg /dns/foo.com/tcp/443/https`,
```